test/result_mapper.test.js

```diff
@@ -62,8 +62,9 @@ describe('Map results and hotels', function () {
   it('map_ne_result_to_graphql maps entire NE API result to GraphQL', function (done) {
     var result = sample_packages_result.result[0];
     assert(Object.keys(result.packageOffer.hotel).length > 5);
+    assert(result.type = 'package'); // https://git.io/vrx7l (issues/75)
```

Comment:
One `=` means this will never fail, as it is doing assignment. Needs "threequals" or `assert.equal`.